theano/tensor/tests/test_opt.py

```diff
@@ -111,7 +111,8 @@ def test_double_transpose(self):
         dimshuffle_lift.optimize(g)
         self.assertTrue(str(g) == "[x]")
         # Check stacktrace was copied over correctly after opt was applied
-        self.assertTrue(check_stack_trace(g, ops_to_check='all'))
+        self.assertTrue(check_stack_trace(g, ops_to_check='all',
+                                          bug_print='ignore'))
```

Comment:
Here, the optimization _removed_ some nodes, without inserting new ones. Then another optimization inserted a DeepCopyOp, but this is not what we want to check here. So you have two options:
- call `check_stack_trace(g, ops_to_check=DimShuffle, bug_print='ignore')`, since the only Op we want to check here is DimShuffle, and it will probably not be there
- not call `check_stack_trace` at all, since the graph is supposed to be empty, and leave a comment explaining that.